src/components/ButtonMenu.stories.tsx

```diff
@@ -157,3 +157,21 @@ export function WithTooltip() {
 
   return <ButtonMenu trigger={{ label: "Menu trigger" }} items={menuItems} tooltip="Tool tip text" />;
 }
+
+export function WithSearchListItems() {
+  const menuItems: MenuItem[] = [
+    { label: "Development 1 - Cohort 1", onClick: action("Development 1 - Cohort 1") },
+    { label: "Development 1 - Cohort 2", onClick: action("Development 1 - Cohort 2") },
+    { label: "Development 1 - Cohort 3", onClick: action("Development 1 - Cohort 3") },
+    { label: "Development 2 - Cohort 1", onClick: action("Development 2 - Cohort 1") },
+  ];
+
+  return (
+    <ButtonMenu
```

Comment:
thought (non-blocking): I wonder if it would be worth having an example of the icon button menu, since that is what we are using.

Reply:
This makes sense, let me update that